diagnostic.md

```diff
@@ -35,5 +39,5 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{my-contact/my-phone contact=contact}}
```

Comment:
close but you need to iterate {{#each contact.phones as |phone|}}  and the {{/each}} after.